dts/arm/st/l4/stm32l4s5.dtsi

```diff
@@ -0,0 +1,235 @@
+/*
+ * Copyright (c) 2020 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <st/l4/stm32l4.dtsi>
```

Comment:
After check, it seems stm32l4s5 is a stm32l4r5 + AES.
Since stm32l4r5 is already available, please use:
`#include <st/l4/stm32l4r5.dtsi>`